apps/src/templates/VisualizationOverlay.js

```diff
@@ -3,8 +3,8 @@
 
 var constants = require('../constants');
 var CrosshairOverlay = require('./CrosshairOverlay');
-var gridUtils = require('./gridUtils');
-var elementUtils = require('./designElements/elementUtils');
+var gridUtils = require('../applab/gridUtils');
+var elementUtils = require('../applab/designElements/elementUtils');
```

Comment:
Does this mean that gamelab now indirectly depends on files in the applab directory? (that seems strange - and a little unexpected)